### PR TITLE
blog/replicating-fama-french-5-factor-model

### PR DESCRIPTION
**Pull request by christophscheuch**
In order to replicate the 5 FF factors, I had to do the following:

- Extend Compustat query & variable construction
- Change Fama-French data download
- Adapt the whole replicating-fama-and-french-factors chapter

I currently only did the last part in R because I want to get approval first. 

The following parts still need to be done:

- Adapt python/replicating-fama-and-french-factors
- Update r/changelog

### CODE REVIEW

File: blog/fama-french-three-vs-five-factors/index.qmd
Comment (patrick-weiss) [inline blog/fama-french-three-vs-five-factors/index.qmd:17]: Maybe add the two paper citations as footnotes.

Would it make sense to investigate Google search requests for the differences between the factors for SEO?

**Review by patrick-weiss — CHANGES_REQUESTED** [review]
Some small suggestions and a potential extension. Great post overall.

File: blog/fama-french-three-vs-five-factors/index.qmd
Comment (christophscheuch) [reply blog/fama-french-three-vs-five-factors/index.qmd:79]: Yeah, I mixed up size and value while I wrote the blog post. The second argument stands, but the first one should go, right?

Comment (christophscheuch) [reply blog/fama-french-three-vs-five-factors/index.qmd:79]: I moved the second argument to the top and adjusted the other argument.

File: blog/fama-french-three-vs-five-factors/index.qmd
Comment (christophscheuch) [reply blog/fama-french-three-vs-five-factors/index.qmd:18]: We hardly get any impressions for fama french replication, so I'd not overinvest for now.

**Comment by christophscheuch** [comment]
From my side the PR is ready to merge, waiting for approval from @patrick-weiss. I'll update changelog after merge and reference the PR.

**Pull request merged**
